lib/hubspot_client/model/contact.rb

```diff
@@ -70,6 +70,12 @@ def assign_attributes(attributes)
           self[attribute] = value
         end
       end
+
+      private
+
+      def writable_properties
+        HubspotClient::Service::Properties.writable_property_names_for('contacts')
```

Comment:
What do you think about cache the result inside the instance, very small performance thing:

```suggestion
        @writable_properties ||= HubspotClient::Service::Properties.writable_property_names_for('contacts')
```